src/batchimages.jl

```diff
@@ -283,21 +288,48 @@ function _is_batch_app(app::DefaultApp)
     compute_type in ("batch", "singlenode-batch") && (input_type == "userinput")
 end
 
+function _is_interactive_batch_app(app::DefaultApp)
+    # Like _is_batch_app, this should return false for JuliaHub <= 6.1
+    compute_type = get(app._json, "compute_type_name", nothing)
+    input_type = get(app._json, "input_type_name", nothing)
+    compute_type in ("distributed-interactive",) && (input_type == "userinput")
+end
+
 function _batchimages_62(auth::Authentication)
     image_groups = _product_image_groups(auth)
-    batchapps = filter(_is_batch_app, _apps_default(auth))
+    batchapps, interactiveapps = let apps = _apps_default(auth)
+        filter(_is_batch_app, apps), filter(_is_interactive_batch_app, apps)
+    end
     batchimages = map(batchapps) do app
         product_name = app._json["product_name"]
         image_group = app._json["image_group"]
         images = get(image_groups, image_group, [])
         if isempty(images)
             @warn "Invalid image_group '$image_group' for '$product_name'" app
         end
+        matching_interactive_app = filter(interactiveapps) do app
+            get(app._json, "image_group", nothing) == image_group
+        end
+        interactive_product_name = if length(matching_interactive_app) > 1
+            # If there are multiple interactive products configured for a batch product
+            # we issue a warning and disable the 'interactive' compute for it (i.e. the user
+            # won't be able to start jobs that require a port to be exposed until the configuration
+            # issue is resolved).
+            @warn "Multiple matching interactive apps for $(app)" image_group matches = matching_interactive_app
```

Comment:
Is this an actual configuration issue? Or should we make users can still choose one of the images manually?

Reply:
Yes, I'd say so. The _images_ are actually the same for the products (same `image_group`), but there are multiple products that seem to do the same thing.